tests/html/bluetooth_device_disconnect.html

```diff
@@ -50,9 +50,8 @@
         try {
             log('Disconnecting from Bluetooth Device...');
             if (bluetoothDevice.gatt.connected) {
-                sleepFor(5000);
```

Comment:
previously I placed the sleepFor() to the wrong line, so I moved underneath the disconnect() method, because we have to wait for the disconnect() to be executed